.github/workflows/ci.yaml

```diff
@@ -101,10 +105,17 @@ jobs:
             cargo-cache-
 
       - name: Run command
+        if: ${{ github.ref != 'refs/heads/main' }}
         run: |
           ./scripts/docker_run ./scripts/runner --commands ${{ matrix.cmd }}
           df --human-readable
 
+      - name: Run Command - Post-Merge
+        if: ${{ github.ref == 'refs/heads/main' }}
```

Comment:
In reproducibility.yaml we use `if: github.event_name == 'push'` for this, could you make sure they are consistent (not sure if one is preferable or they are the same)

Reply:
Thanks. I'll try this one.